doc/reference/stability.rst

```diff
@@ -0,0 +1,206 @@
+.. _api_stability:
+
+API Stability
+#############
+
+The table below lists the available documented APIs in Zephyr, their
+:ref:`current status <api_lifecycle>` and the release that introduced them.
+
+.. list-table::
+   :header-rows: 1
+
+   * - API
+     - Status
+     - Version Introduced
+
+   * - :ref:`audio_codec_api`
+     - Experimental
+     - 1.13
+
+   * - :ref:`audio_dmic_api`
+     - Experimental
+     - 1.13
+
+   * - :ref:`bluetooth_api`
+     - Stable
+     - 1.0
+
+   * - :ref:`can_api`
+     - Experimental
+     - 1.14
+
+   * - :ref:`crypto_api`
+     - Stable
+     - 1.7
+
+   * - :ref:`device_model_api`
+     - Stable
+     - 1.0
+
+   * - :ref:`display_api`
+     - Unstable
+     - 1.14
+
+   * - :ref:`file_system_api`
+     - Stable
+     - 1.5
+
+   * - :ref:`kernel_api`
+     - Stable
+     - 1.0
+
+   * - :ref:`logging_api`
+     - Stable
+     - 1.13
+
+   * - :ref:`misc_api`
+     - Stable
+     - 1.0
+
+   * - :ref:`networking_api`
+     - Stable
+     - 1.0
+
+   * - :ref:`adc_api`
+     - Stable
+     - 1.0
+
+   * - :ref:`clock_control_api`
+     - Stable
+     - 1.0
+
+   * - :ref:`counter_api`
+     - Unstable
+     - 1.14
+
+   * - :ref:`dma_api`
+     - Stable
+     - 1.5
+
+   * - :ref:`eeprom_api`
+     - Experimental
+     - 2.1
+
+   * - :ref:`entropy_api`
+     - Stable
+     - 1.10
+
+   * - :ref:`flash_api`
+     - Stable
+     - 1.2
+
+   * - :ref:`gna_api`
+     - Experimental
+     - 1.14
+
+   * - :ref:`gpio_api`
+     - Stable
+     - 1.0
+
+   * - :ref:`hwinfo_api`
+     - Unstable
```

Comment:
I would like to declare hwinfo as stable.

Reply:
This will be coming soon